com.unity.ml-agents/Runtime/Academy.cs

```diff
@@ -30,6 +30,15 @@ namespace Unity.MLAgents
     /// </summary>
     internal class AcademyFixedUpdateStepper : MonoBehaviour
     {
+        void OnEnable()
+        {
+            // Check if this stepper belongs to current Academy singleton
+            // if (!Academy.Instance.IsStepperOwner(this))
```

Comment:
Uncomment?
Just to be extra cautious, I think you should either
1) Change the condition to `if(Academy.IsInitialized && !Academy.Instance.IsStepperOwner(this))`
2) Make IsStepperOwner static and check `Academy.IsInitialized` there
Just so that we're not accidentally creating an AcademyInstance when we don't want to  (which is what got us into this in the first place).